DigitalLearningSolutions.Data.Migrations/RecurringSqlJobs/DeleteMultipageFormDataOlderThan30Days.sql

```diff
@@ -0,0 +1,4 @@
+﻿-- Deletes MultiPageFormData records older than 30 days. Should be run every night at 2AM.
+DELETE
+FROM MultiPageFormData
+WHERE CAST(CreatedDate AS DATE) <= GETDATE() - 30

```

Comment:
I presume we are casting to DATE here so that we aren't concerned about the hours/minutes/seconds. Is that something we are sure we want? Bit confusing as GETDATE also returns a DATETIME, which we have just cast away from on the CreatedDate.

I don't _think_ there are any scenarios where ignoring the hours/minutes/seconds will be important, but thought it is worth double checking.

Reply:
Yeah, I agree that the casting is strange here. I would have done `WHERE CreatedDate < DATEADD(day, -30, GETDATE())`
It feels wrong to me to just have a `-30` -30 what? Days? Minutes?

Reply:
Makes sense to me, I've gone with your suggestion, Alex.